### PR TITLE
📖 Bookmark Mentors Sidebar

### LINKED ISSUE #293: Mentee portal - bookmark mentors side bar
image

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

Fixes #293
- Implemented base design of the buttons
- Still needs the functionality to be implemented
- Depends #299 
 
## Screenshots
image
image

## TODOs :star:
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: frontend/src/components/BookmarkSidebar.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        {bookmarks.map((mentor) => (
+          <div className="mentee-bookmark-card">
+            <Avatar
+              src={mentor && mentor.image.url}
+              icon={<UserOutlined />}
+              size={30}
+              style={{ minWidth: "30px" }}
+            />
+            <div className="mentee-bookmark-mentor-info">
+              <HeartFilled className="bookmark-heart" />
```
Comment (kenetec) [inline frontend/src/components/BookmarkSidebar.js:29]: On click, should unfavorite?

**Review by kenetec — APPROVED** [review]
Awesome work!! 🤩 make sure the svg is minified!

File: frontend/src/components/BookmarkSidebar.js (hunk 1 above)
Comment (AngelaLuo49021) [reply frontend/src/components/BookmarkSidebar.js:29]: yuh

**Review by AngelaLuo49021 — APPROVED** [review]
Overall looks great! Please open an issue for anything missing: functionality, backend connection etc.

**Pull request merged**